server/metadata/dictionary.go

```diff
@@ -56,20 +56,24 @@ import (
 // Request: To Create a New Database
 // The value would be derived from the counter value stored in the reserved subspace. If the counter is not yet issued
 // then it will start from 0x01.
-//    ["encoding", 0x01, x "db", "db-1", "created"] = 0x01
-//    ["encoding", 0x01, x, "db", "db-2", "created"] = 0x02
+//
+//	["encoding", 0x01, x "db", "db-1", "created"] = 0x01
```

Comment:
What's wrong with original alignment?

Reply:
nothing is/was wrong. IDE modified it. We at some point need some shared IDE settings.